### ISSUE TITLE
analyze() and init() fail

### ISSUE THREAD

**Issue by some-username** [radare/r2pipe.rs#15]
For my r2 version
````
$ r2 -v
radare2 1.0.2 9999999 @ linux-x86-64 git.1.0.2
commit: HEAD build: 2016-11-11
````
calling `r2.send("aaaaa")` with more than 3 `"a"`s fails with emitting lots of
````
r_io_write: cannot write 1 bytes at 0x7f (file=/home/..., fd=6)
hint: try oo+ or e io.cache=true
TRAP
````
`analyze()` (called by `init()`) calls `send()` with `"aaaaaaaa"`. Thus both fail to work.

**Comment by some-username** [comment]
Simple fix:
Change `src/rs.s`:164 from
````
self.send("aaaaaaaa");
````
to
````
self.send("aaa");
````

**Comment by sushant94** [comment]
Fixed in fdd1c5a
